lib/commons/dom/is-visible.js

```diff
@@ -133,7 +136,10 @@ function isVisible(el, screenReader, recursed) {
 		 */
 		(nodeName !== 'AREA' && style.getPropertyValue('display') === 'none') ||
 		['STYLE', 'SCRIPT', 'NOSCRIPT', 'TEMPLATE'].includes(nodeName) ||
-		(!screenReader && isClipped(style)) ||
+		(!screenReader &&
+			(isClipped(style) ||
+				style.getPropertyValue('opacity') === '0' ||
+				(getScroll(el) && parseInt(style.getPropertyValue('height')) === 0))) ||
```

Comment:
We need to clean up this if statement. A 10 line if condition seems a bit much, no?